_posts/2019-11-12-multiline-javascript-converter.md

```diff
@@ -7,25 +7,25 @@ categories: projects
 
 <p class="center">
     <img src="https://raw.githubusercontent.com/bbody/MultilineJavaScriptConverter/master/screenshots/demo.gif" alt="Demo of Multiline JavaScript Converter" />
 </p>
 
-Although the project was started in 2016, in 2018 I picked it up again. Unlike [CMD Resume](https://github.com/bbody/CMD-Resume/), I severely limited the scope. This lead to using Mocha/Chai and Nightwatch for testing to try different testing frameworks. Instead of trying to test a broad range of browsers, I only tested UI on the CI server using Chromium. Combined with using ESLint, the verification process is very quick and easy. Originally I used Travis for CI however more recently I switched to Github Actions, which was an interesting experience.
+Although the project was started in 2016, I picked it up again in 2018. Unlike [CMD Resume](https://github.com/bbody/CMD-Resume/), I severely limited the scope. This led to using Mocha/Chai and Nightwatch for testing to try different testing frameworks. Instead of trying to test a broad range of browsers, I only tested the UI on the CI server using Chromium. Combined with ESLint, the verification process became very quick and easy. Originally, I used Travis for CI; however, more recently I switched to GitHub Actions, which was an interesting experience.
 
-Although this project is not as complex as CMD Resume, I did manage to learn a bit. I tried a few new frameworks, particularly Mocha/Chai which I'd been itching to try for a little while. Whilst introducing tests, due to the size of the project I was able to decouple logic from the UI using Browserify and was able to write better tests.
+Although this project is not as complex as CMD Resume, I did manage to learn quite a bit. I tried a few new frameworks, particularly Mocha/Chai, which I'd been itching to try for a while. While introducing tests, due to the project's manageable size, I was able to decouple logic from the UI using Browserify and write better tests.
 
-I do think I may have spent more time than was worth it on Multiline JavaScript Convertor. Particularly as I can use ECMA6 in most of my products, however I have used it for ASCII art in CMD Resume. I also went down the track of turning it into a [NPM package](https://www.npmjs.com/package/multilinejavascriptconverter), which leveraged the decoupling of logic and UI. The problem is that the other project I planned to use it for got scrapped.
+I do think I may have spent more time than was worthwhile on Multiline JavaScript Converter. This is particularly true since I can use ECMA6 in most of my projects now; however, I have found use for it with ASCII art in CMD Resume. I also went down the path of turning it into an [NPM package](https://www.npmjs.com/package/multilinejavascriptconverter), which leveraged the decoupling of logic and UI. The problem is that the other project I planned to use it for got scrapped.
```

Comment:
```suggestion
I do think I may have spent more time than was worthwhile on Multiline JavaScript Converter. This is particularly true since I can use ECMA6 in most of my projects now, however, I have found use for it with ASCII art in CMD Resume. I also went down the path of turning it into an [NPM package](https://www.npmjs.com/package/multilinejavascriptconverter), which leveraged the decoupling of logic and UI. The problem is that the other project I planned to use it for got scrapped.
```